components/metal/MetalGrid.vue

```diff
@@ -34,158 +34,143 @@
+        (isInDecoherenceLeftColumnZone && leftColumnIsDecoherent) ||
+        (isInDecoherenceRightColumnZone && rightColumnIsDecoherent)
+
+      yield { c, r: currentRow, isDecoherent }
     }
+  }
 
-    return generateRows()
-  })())
-
-  fallingCells: string[] = Array.from((() => {
-    const { noDecoherenceColumnCount, pattern } = this
-    const self = this
-    const centralColumn = Math.floor(noDecoherenceColumnCount / 2)
-    const [rowStart, rowEnd] = [1, pattern.length + 1]
-    const [columnStart, columnEnd] = [
-      centralColumn - pattern[0].length / 2,
-      centralColumn + pattern[0].length / 2
-    ]
-
-    function* gen (): Iterable<string> {
-      for (let r = rowStart; r < rowEnd; r++) {
-        for (let c = columnStart; c < columnEnd; c++) {
-          const cellIsAlwaysVisible = pattern[r - rowStart][c - columnStart] === 0
-          if (cellIsAlwaysVisible) {
-            continue
-          }
-          yield self.posId({ c, r })
+  return generateRows()
+})()))
+
+const fallingCells = computed<string[]>(() => Array.from((() => {
+  const self = this
+  const centralColumn = Math.floor(noDecoherenceColumnCount / 2)
+  const [rowStart, rowEnd] = [1, pattern.length + 1]
+  const [columnStart, columnEnd] = [
+    centralColumn - pattern[0].length / 2,
+    centralColumn + pattern[0].length / 2
+  ]
+
+  function* gen (): Iterable<string> {
+    for (let r = rowStart; r < rowEnd; r++) {
+      for (let c = columnStart; c < columnEnd; c++) {
+        const cellIsAlwaysVisible = pattern[r - rowStart][c - columnStart] === 0
+        if (cellIsAlwaysVisible) {
+          continue
         }
+        yield self.posId({ c, r })
       }
     }
-
-    return gen()
-  })())
-
-  isHidden (pos: CellSpecification) {
-    return this.hiddenCells.includes(this.posId(pos))
   }
 
-  isTrigger (pos: CellSpecification) {
-    const centralColumn = Math.floor(this.noDecoherenceColumnCount / 2)
-    const { x: triggerX, y: triggerY } = this.triggerPositionFromTopCenter
-    const { c, r } = pos
-    return c === centralColumn + triggerX && r === triggerY
-  }
+  return gen()
+})()))
 
-  triggerAnimation (pos: CellSpecification) {
-    if (!this.isTrigger(pos)) { return }
-    this.slotContainerIsHidden = true
-    this.removeCell()
-  }
+const isHidden = computed((pos: CellSpecification) => {
+  return hiddenCells.includes(posId(pos))
+})
 
-  removeCell () {
-    const length = this.fallingCells.length
-    const noMoreCells = length === 0
+const isTrigger = computed((pos: CellSpecification) => {
+  const centralColumn = Math.floor(noDecoherenceColumnCount / 2)
+  const { x: triggerX, y: triggerY } = triggerPositionFromTopCenter
+  const { c, r } = pos
+  return c === centralColumn + triggerX && r === triggerY
+})
 
-    if (noMoreCells) {
-      setTimeout(() => {
-        this.$router.push({ path: '/metal' })
-      }, this.timeToLoadMetal)
-      return
-    }
+const triggerAnimation = (pos: CellSpecification) => {
```

Comment:
We are using `function xxxx () {}` instead of `const xxxx = () => {}` . Is there any reason you need this second syntax here? I don't mind using one type or another, but it would be nice to follow a convention

Reply:
Agree. Changed in https://github.com/Qiskit/qiskit.org/pull/2935/commits/8528b816e07a7f7e71d0860733ba1431cdb149ba.